src/foam/u2/view/TreeView.js

```diff
@@ -63,6 +61,10 @@ foam.CLASS({
       border-left: 4px solid rgba(0,0,0,0);
     }
 
+    ^select-level {
+      padding: 8px;
+    }
+
     ^selected > ^label {
       xxxborder-radius: 2px;
```

Comment:
Remove this line and next which aren't needed anymore.

Reply:
fixed

Reply:
fixed